objectRecognitionSource/README.md

```diff
@@ -167,6 +167,12 @@ The Configuration document may look similar to the following example:
              "outputDir": "imageOut",
              "savedResolution": {
                 "longEdge": 400
+             },
+             "preCrop": {
```

Comment:
Prefer to have something more descriptive like `cropBeforeAnalysis`.  (re-mentioned in the code which I started before the readme.